src/legacy/core_plugins/kibana/public/dashboard/dashboard_app.js

```diff
@@ -122,6 +122,8 @@ app.directive('dashboardApp', function ($injector) {
 
       $scope.getDashboardState = () => dashboardStateManager;
       $scope.appState = dashboardStateManager.getAppState();
+      $scope.refreshInterval = timefilter.getRefreshInterval();
```

Comment:
Hey @kertal - I'm working on merging this change into a PR.  In my PR, which I typescripted things, I don't actually have a refreshInterval on scope.  Are there tests to ensure that will catch if I incorrectly merge this?

I do see $scope.modal.refreshInterval...  or maybe I just need to add a refreshInterval on scope?

Reply:
Hi @stacey-gammon - sorry I somehow missed this comment. well, there is no test, but there should be a functional test to prevent a comeback of this issue, right? 

Having a quick look at the code, I'd assume $scope.model.refreshInterval should be set, I'll analyze it, and let you know

Reply:
> but there should be a functional test to prevent a comeback of this issue, right?

I'm assuming no because all the functional tests were passing prior to this PR? Unless I missing that?  But it would be great to have one added that catches whatever bug this fixed (sorry, I didn't dig too deeply)

Reply:
yes, there were no functional tests, but I will create one for each of Discover, Visualize and Dashboard

Reply:
awesome, thanks!! 🙇‍♀